galaxy.yml

```diff
@@ -0,0 +1,11 @@
+---
+# This file is needed to run some tests. The action does not need this,
```

Comment:
How about having this file somewhere under "tests" or "artifacts" and copying it in that one job running sanity?